autogpt/llm_utils.py

```diff
@@ -83,7 +83,7 @@ def create_chat_completion(
         try:
             if CFG.use_azure:
                 response = openai.ChatCompletion.create(
-                    deployment_id=CFG.get_azure_deployment_id_for_model(model),
+                    engine=CFG.get_azure_deployment_id_for_model(model),
```

Comment:
Seems to be in accordance with the docs https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/chatgpt?pivots=programming-language-chat-completions